arbnode/batch_poster.go

```diff
@@ -557,7 +557,15 @@ func AccessList(opts *AccessListOpts) types.AccessList {
 func (b *BatchPoster) checkEspressoValidation(
 	msg *arbostypes.MessageWithMetadata,
 ) error {
-	if !b.streamer.chainConfig.ArbitrumChainParams.EnableEspresso {
+	arbOSConfig, err := b.arbOSVersionGetter.GetArbOSConfigAtHeight(0)
```

Comment:
why are we only checking it at height 0?

Reply:
`0` means fetching the config from the latest height